app/controllers/landmarks_controller.rb

```diff
@@ -10,6 +10,7 @@ def index
   def show
     @landmark = Landmark.find(params[:id])
     @review = Review.new
+    @landmark.average_review = @landmark.average_rating
```

Comment:
I think you could just run the method as part of displaying the `show` page and then call the storage of that value in the view, if you wanted to shorten up this method a bit.